datadog/resource_datadog_dashboard.go

```diff
@@ -3507,6 +3507,7 @@ func getQueryValueRequestSchema() map[string]*schema.Schema {
 		"process_query":  getProcessQuerySchema(),
 		"rum_query":      getApmLogNetworkRumSecurityQuerySchema(),
 		"security_query": getApmLogNetworkRumSecurityQuerySchema(),
+		"audit_query":    getApmLogNetworkRumSecurityQuerySchema(),
```

Comment:
For the docs change, you'll have to edit the description in the schema here, because docs are generated from them

Reply:
> > Docs review
> 
> @ruthnaebeck I tried to commit your suggested changes, but I think the CI tests do not allow me to make manual changes to auto-generated docs. So I think the wording cannot be changed.

Ok. Can we update the schema for docs standards? I can make updates to your branch if needed.